source/CollisionSet.cpp

```diff
@@ -166,8 +166,31 @@ Body *CollisionSet::Line(const Point &from, const Point &to, double *closestHit,
 	// Keep track of the closest collision found so far. If an external "closest
 	// hit" value was given, there is no need to check collisions farther out
 	// than that.
-	double closest = closestHit ? *closestHit : 1.;
-	Body *result = nullptr;
+	class Closest {
```

Comment:
Can you move this class into the anonymous namespace at the top?

Reply:
Done